### ISSUE TITLE
Not compatible with DRF==3.10.1

### ISSUE THREAD

**Issue by PetrDlouhy** [jozo/djangorestframework-camel-case2#9]
After upgrade to `django-rest-framework==3.10.1` I got following error:
```
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/petr/soubory/programovani/blenderhub_server/blenderhub/urls.py", line 9, in <module>
    from wagtail.admin import urls as wagtailadmin_urls
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/wagtail/admin/urls/__init__.py", line 13, in <module>
    from wagtail.admin.api import urls as api_urls
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/wagtail/admin/api/urls.py", line 6, in <module>
    from .endpoints import PagesAdminAPIEndpoint
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/wagtail/admin/api/endpoints.py", line 3, in <module>
    from wagtail.api.v2.endpoints import PagesAPIEndpoint
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/wagtail/api/v2/endpoints.py", line 12, in <module>
    from rest_framework.viewsets import GenericViewSet
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/viewsets.py", line 27, in <module>
    from rest_framework import generics, mixins, views
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/generics.py", line 9, in <module>
    from rest_framework import mixins, views
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/views.py", line 104, in <module>
    class APIView(View):
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/views.py", line 108, in APIView
    parser_classes = api_settings.DEFAULT_PARSER_CLASSES
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/settings.py", line 220, in __getattr__
    val = perform_import(val, attr)
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/settings.py", line 168, in perform_import
    return [import_from_string(item, setting_name) for item in val]
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/settings.py", line 168, in <listcomp>
    return [import_from_string(item, setting_name) for item in val]
  File "/home/petr/.local/share/virtualenvs/blenderhub_server-izvs0qj4/lib/python3.6/site-packages/rest_framework/settings.py", line 180, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'djangorestframework_camel_case2.parser.CamelCaseJSONParser' for API setting 'DEFAULT_PARSER_CLASSES'. ImportError: cannot import name 'six'.

```

**Comment by jozo** [comment]
It was probably just a problem with bad import. I fixed it and released new version (0.2.4). Can you try it if it works? @PetrDlouhy

**Comment by PetrDlouhy** [comment]
@jozo Thanks. Everything seems to work fine.
